gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=3.0.5
+version=3.0.6
```

Comment:
Last, this seems not rebased. Master has already been 3.0.6

Reply:
Thanks! Rebased.